config.yaml

```diff
@@ -288,7 +288,16 @@ options:
       profile will be extracted from '/var/log/cloud-init-output.log', fallback is 'level1_server'.
       Options are: '' (disable profile check), 'level1_server', 'level2_server',
                    'level1_workstation' or 'level2_workstation'
+      This option cannot be used with cis_audit_tailoringfile at the same time.
       See also https://ubuntu.com/security/certifications/docs/cis-audit
+  cis_audit_tailoringfile:
```

Comment:
**suggestion (non-blocking):** 

Should we use base64 to encode the content instead using plain text?

Reply:
We had a [discussion](https://chat.canonical.com/canonical/pl/h6mhuwtgpi8ujeztege16dfy3r) if this should be a normal string or base64 and we decided to keep as normal string. We didn't see to much advantages turning into base64. The truth is that juju was not made to pass a file as configuration